sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -73,61 +73,69 @@ add_sycl_plugin(unified_runtime
     Threads::Threads
     UnifiedRuntimeLoader
     UnifiedRuntime-Headers
-    LevelZeroLoader-Headers # we need for #include <ze_api.h> in common.h
```

Comment:
No `common.h` in sight, and nothing in `sycl/plugins/unified_runtime/` rely on level_zero except the corresponding adapter.